.circleci/cluster/manager1_config.yaml

```diff
@@ -19,9 +21,15 @@ postgresql_server:
   postgres_password: 'somesecretvalue123!'
   cluster:
     nodes:
-      - DB1_IP
-      - DB2_IP
-      - DB3_IP
+      db_1:
+        ip: DB1_IP
+        node_id: 'db1_node_id'
```

Comment:
Same question as for rabbit.

Reply:
I want to have also the hostname, I'll use the hostname in the response of the cluster status.